### PR TITLE
Make changes to support early access version

### PR DESCRIPTION
**Pull request by HansNewbie**

### CODE REVIEW

**Comment by damithc** [comment]
Also make sure,
1. Our version numbers comply with `semantic versioning` conventions
2. Some basic documentation

**Comment by HansNewbie** [comment]
@damithc noted.

File: src/main/java/address/util/Version.java [hunk 1]
```diff
@@ -53,25 +59,40 @@ public static Version fromString(String versionString) throws IllegalArgumentExc
...
     }
 
     @JsonValue
     public String toString() {
-        return String.format("V%d.%d.%d", major, minor, patch);
+        if (isEarlyAccess) {
+            return String.format("V%d.%d.%dea", major, minor, patch);
+        } else {
+            return String.format("V%d.%d.%d", major, minor, patch);
+        }
```
Comment (yl-coder) [inline src/main/java/address/util/Version.java]: Can reduce this block to one line

return String.format("V%d.%d.%d%s", major, minor, patch, isEarlyAccess ? "ea" : "");

**Comment by HansNewbie** [comment]
Will merge if no more comments.

**Pull request merged**